doc/Internals.md

```diff
@@ -460,5 +460,7 @@ So if you need to trigger events in theia:
  - Register events with the emitter.event function
  - Fire events with emitter.fire(event)
 
+# Preferences
 
+You can check out [this page](Preferences.md) to learn more about how preferences work within Theia.
 
```

Comment:
Good point, and I even think that the elements currently in that list should be split into their own files.